src/controller/sql/mod.rs

```diff
@@ -618,6 +618,8 @@ impl SqlIncorporator {
             // other kinds of queries *do* require their referred tables to exist!
             ref q @ SqlQuery::CompoundSelect(_) |
             ref q @ SqlQuery::Select(_) |
+            ref q @ SqlQuery::Update(_) |
```

Comment:
Let me know how you want to handle the new `Update` and `Delete` expressions. I added `unreachable!` in a couple of other places, but included them here since it seemed to make sense.

Reply:
Yes, this is the right way. `unreachable!` would be fine here too, but this is correct if we ever choose to support them later.